src/android/BiometricActivity.java

```diff
@@ -16,15 +16,19 @@
 
 import java.util.concurrent.Executor;
 
+import javax.crypto.Cipher;
+
 public class BiometricActivity extends AppCompatActivity {
 
     private static final int REQUEST_CODE_CONFIRM_DEVICE_CREDENTIALS = 2;
     private PromptInfo mPromptInfo;
+    private CryptographyManager mCryptographyManager;
+    private static final String SECRET_KEY = "__aio_secret_key";
```

Comment:
Should we allow changing this secret key via a parameter?

Reply:
I don't think changing the name of the suffix would be needed, as the SECRET_KEY is only to complement and store more safely on the encryption data.